subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/radio/radio.c

```diff
@@ -26,13 +26,15 @@
 #include "hal/swi.h"
 #include "fsl_cau3_ble.h"	/* must be after irq.h */
 
-#define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_DRIVER)
-#define LOG_MODULE_NAME bt_openisa_radio
-#include "common/log.h"
 #include "common/assert.h"
+
 #include <soc.h>
 #include "hal/debug.h"
 
+#define LOG_LEVEL CONFIG_BT_HCI_DRIVER_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(bt_openisa_radio);
```

Comment:
no need to define `LOG_LEVEL` just put it as second argument to the macro.

Reply:
This won't work when `CONFIG_BT_HCI_DRIVER_LOG_LEVEL` is not defined, which unfortunately can happen right now.

If `CONFIG_BT_HCI_DRIVER_LOG_LEVEL` is placed as the second argument of `LOG_MODULE_REGISTER` but `CONFIG_BT_HCI_DRIVER_LOG_LEVEL` is undefined, it will result in `LOG_MODULE_REGISTER` having an empty second argument which is not allowed.

It's already done like that in others subsystems, here are some examples:
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/usb/device/os_desc.c#L7
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/net/buf.c#L10
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/net/lib/lwm2m/lwm2m_message_handling.c#L16
https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/emul/i2c/emul_sbs_gauge.c#L11

Reply:
I see, so there is a file that is compiled but it uses a level from module which might be disabled (and because of that level is not available)? It looks as if some dependency cleanup would be needed but as i understand this PR does not intend to change anything.